src/map/script.c

```diff
@@ -20723,6 +20724,82 @@ BUILDIN(mergeitem)
 	return true;
 }
 
+// getcalendartime(<day of month>, <day of week>{, <hour>{, <minute>}});
+// Returns the UNIX Timestamp of next ocurrency of given time
+BUILDIN(getcalendartime)
+{
+	struct tm info = { 0 };
+	int day_of_month = script_hasdata(st, 4) ? script_getnum(st, 4) : -1,
+		day_of_week = script_hasdata(st, 5) ? script_getnum(st, 5) : -1,
+		year = date_get_year(),
+		month = date_get_month(),
+		day = date_get_day(),
+		hour = script_getnum(st, 2),
+		minute = script_getnum(st, 3);
```

Comment:
Would you mind splitting these into separate declarations (add `int` to each line)? I find it cleaner than using commas and indentation, when multiple lines are needed